### PR TITLE
Fix typos and line ending chars

### PR DESCRIPTION
**Pull request by rejahrehim**
- [x] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Fix typos

### CODE REVIEW

**Comment by kingthorin** [comment]
What was the EOL change CRLF to LF?

**Comment by rejahrehim** [comment]
> What was the EOL change CRLF to LF?

Yes. It was getting some error while trying to run PDF build.

File: document/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 ### [2.3 Testing Techniques Explained](2-Introduction/README.md#Testing-Techniques-Explained)
 
-### [2.4 Manual Inspections & Reviews](2-Introduction/README.md#Manual-Inspections-and-Reviews)
+### [2.4 Manual Inspections and Reviews](2-Introduction/README.md#Manual-Inspections-and-Reviews)
```
Comment (rejahrehim) [inline document/README.md:17]: To make the header and toc alike.

File: document/4-Web_Application_Security_Testing/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Web Application Penetration Testing
+# Web Application Security Testing
```
Comment (rejahrehim) [inline document/4-Web_Application_Security_Testing/README.md:1]: To make header and TOC alike

**Pull request merged**